FEM-2489 - Artwork view visibility logic with playkit ExoPlayerView

GouravSna opened the pull request:
- Change in IMAPlugin for AdEvent logic
- When ads are visible, with the video surface view hide the artwork view.

GouravSna committed:
- Artwork view visibility logic with playkit ExoPlayerView

giladna reviewed (approved):
please wait with merge until we merge the playkit

GouravSna committed:
- Renamed `artworkViewVisibility` to `setArtworkViewVisibility`

GouravSna committed:
- Removed artwork view logic from Ads.
- Now the logic is being handled in PlayerController and ExoPlayerView inside Playkit itself.
